draft-sharabayko-mops-srt.md

```diff
@@ -2057,127 +2057,586 @@ i.e., carried by an incoming ACK.
+on the input rate and an overhead for packets retransmission in order
 to avoid congestion during fluctuations of the source bitrate.
-LiveCC determines the minimum time between consecutive packets are sent.
-Also, it is allowed to drop those packets that can not be delivered in time.
 
-On the receiving side, LiveCC may decide when an ACK is needed prior to ACK timeout.
-It also determines the interval of periodic NAK packets to be reported to the sender.
+In addition to these mechanisms, in highly variable networks live mode
+fairness can be accomplished outside SRT by controlling the bitrate
+of the encoder at the input of the SRT sender. SRT provides a variaty
+of network related statistics such as RTT estimation, the level of
+packet loss, the number of dropped by SRT packets, etc., which can be
+used for making decisions and adjusting the bitrate in real time.
 
-### File Transfer Congestion Control (FileCC)
+### Configuring Maximum Bandwidth {#maxbw}
 
-For file transfer, any known File Congestion Control algorithms, like CUBIC {{RFC8312}}, can apply,
-including the congestion control mechanism proposed in UDT {{GHG04b}}.
-The UDT congestion control relies on the available link capacity, packet loss reports (NAK)
-and packet acknowledgments (ACKs).
-It then slows down the output of packets as needed by adjusting the packet sending pace.
-In periods of congestion, it can block the main stream and focus on the lost packets.
+There are several ways of configuring maximum bandwidth MAX_BW:
 
-The algorithm consists of three states: slow start, congestion avoidance and slow down.
-Starting with no information, the congestion control module probes the network to
-determine the available bandwidth and the sending pace for the desired operation mode: congestion avoidance.
-In this state, if there is no congestion detected via Loss reports, the sending pace is gradually increased.
-On the other side, if congestion is detected in the network the state is changed to slow down and
-the sending pace is decreased to reduce packet loss.
+1. By means of setting the value of maximum bandwidth MAX_BW
+explicitly (MAXBW mode).
 
-#### Slow Start
+The recommended default value is 1 Gbps. Note that the default value
+is set only for live transmission mode.
 
-Slow start state has a limit on the number of data packets that can be sent before
-acknowledgment of those segments is received. File CC starts with 16 DATA packets (CWND_SIZE=16)
-sent with minimum possible interval.
+Note that this static setting is not well-suited to a variable input,
+like when you change the bitrate on an encoder. Each time the input
+bitrate is configured on the encoder, MAX_BW must also be reconfigured.
+
+In this mode, SRT does not follow the fluctuations in the encoder input
+making it not suitable for variable bitrate (VBR) encoding use cases.
+
+2. By means of setting the SRT sender's input rate INPUT_BW and the
+overhead OVERHEAD (INPUTBW mode).
```

Comment:
### TODO:

- [ ] Fix list enumeration in TXT output (all numbers produce "1").